python/cuda_parallel/cuda/parallel/experimental/iterators/_iterators.py

```diff
@@ -146,21 +160,38 @@ def ltoirs(self, value):
     def state(self) -> IteratorState:
         return self.state_
 
-    @staticmethod
-    def advance(state, distance):
-        raise NotImplementedError("Subclasses must override advance staticmethod")
+    @property
+    def advance(state):
+        raise NotImplementedError("Subclasses must override advance property")
 
-    @staticmethod
-    def dereference(state):
-        raise NotImplementedError("Subclasses must override dereference staticmethod")
+    @property
+    def dereference(state, *args):
```

Comment:
Did `*args` slip in here by accident?

(None of the other `@property` `def dereference` seems to have it.)

Reply:
Yeah I forgot to remove it